requests/OmgUsdtPrice.js

```diff
@@ -0,0 +1,77 @@
+import * as Witnet from "witnet-requests"
+
+// Retrieves USDT price of OMG from the Binance API
+const binance = new Witnet.Source("https://api.binance.com/api/v3/ticker/price?symbol=OMGUSDT")
+  .parseJSONMap() // Parse a `Map` from the retrieved `String`
+  .getFloat("price") // Get the `Float` value associated to the `price` key
+  .multiply(1000000) // Use 6 digit precision
```

Comment:
Let's take this chance to use format `10 ** 6` in this file too. It's less error-prone I think.